.github/workflows/deploy-automatic.yml

```diff
@@ -0,0 +1,28 @@
+# If new commits are available since the last tag, create a new version/tag
+# - Every day at 13:13
```

Comment:
Why the magic time `13:13` though, just curious.

Reply:
Because it's a lucky number!

hehe

Any time works, as long as it's not the "default" midnight (`0 0 * * *`)